### ISSUE TITLE
orchestra-cljs.spec.test/instrument-1* is not public

### ISSUE THREAD

**Issue by miikka** [jeaye/orchestra#27]
The soon-to-be released version of ClojureScript compiler [warns about the use of private variables](https://github.com/clojure/clojurescript-site/blob/news-next/content/news/2018-09-01-release.adoc#warnings-on-private-var-use). Now using `orchestra-cljs.spec.test/instrument` causes a lof of warnings like this:

```
WARNING: var: orchestra-cljs.spec.test/instrument-1* is not public at line 27 src/cljs/frontend/spec.cljs
```

Btw, thanks for Orchestra – I'm now using it everywhere instead of spec's own instrument.

**Comment by conan** [comment]
Warnings prevent figwheel loading code, so this is a slightly tricker issue than just console output.

**Comment by jeaye** [comment]
Thanks, guys. This should be fixed in the `2018.11.07-1` release.
